app/assets/stylesheets/helpers/_core.scss

```diff
@@ -298,38 +298,6 @@ h4 {
   }
 }
 
-#wrapper.businesslink #legacy-sources,
-#wrapper.directgov #legacy-sources {
-  display: block;
-}
-
-#wrapper.businesslink #legacy-sources li.businesslink {
-	background-image: image-url("businesslink-logo.png");
```

Comment:
A bit scared of removing since: https://github.com/alphagov/performanceplatform-selfserve and https://github.com/alphagov/prototyping seem to be referring to it: @alextea @bradleywright what would you advise?